db_connection.py

```diff
@@ -27,25 +32,25 @@ def connect_db(self, connection_parameters):
                     port=connection_parameters['port']
                 )
                 self.cursor = self.conn.cursor()
-                print("Connection Success!")
+                logger.info("Connection Success!")
                 return 
             except OperationalError as e:
                 attempts += 1
-                print(f"Connection error: {e}. Attempt {attempts}/{MAX_CONNECTION_ATTEMPTS}.")
+                logger.error(f"Attempt {attempts}/{MAX_CONNECTION_ATTEMPTS}. Connection error: {e}")
                 time.sleep(DELAY)
             except Exception as ex:
-                print(f'[Err] exception has been caught during establishing connection to db: {ex}')
+                logger.critical(f'[Err] exception has been caught during establishing connection to db: {ex}')
             finally:
                 if 'connection' in locals() and self.conn is not None:
                     self.conn.close()
-        print("Error, max attempts of connection overlay")
+        logger.critical("Error, max attempts of connection overlay")
         raise RuntimeError("max attempts of connection overlay")
 
     def start_docker_client(self):
         try:
             self.docker_client = DockerClient(base_url='unix://var/run/docker.sock')
         except Exception as ex:
-            print(f'[Err] exception has been caught during starting docker client: {ex}')
+            logger.error(f'[Err] exception has been caught during starting docker client: {ex}')
```

Comment:
Same comments as for necessity of [Err] and exc_info=True parameter.

Reply:
resolved